schema.sql

```diff
@@ -1,5 +1,9 @@
 /* Database schema to keep the structure of entire database. */
-
 CREATE TABLE animals (
-    name varchar(100)
-);
+    id int,
```

Comment:
### OPTIONAL

- [ ] It will be better to make the `id` not null and autoincremented. This will ensure for every data in the table there is an `id`.
- [ ] It will be great to mark the `id` column as the **`PRIMARY KEY`**

Reply:
Done!